tests/profile/test_views.py

```diff
@@ -139,3 +130,27 @@ def test_matched_url(self, profile, rf):
         request = rf.get("profile-by-url", HTTP_WM_HOSTNAME="some_domain.com")
         response = ProfileByUrl.as_view()(request)
         assert response.status_code == 200
+
+
+@pytest.mark.django_db
+class TestProfileIndicatorData(
+    APIViewTest,
+    UsesGetMethod,
+    Returns200
+
+):
+
+    @pytest.fixture
+    def url(self, profile, geography, grouped_profile_indicator):
+        return reverse("profile-geography-indicator-data", kwargs={
```

Comment:
you don't need to use `reverse` here. 
we have `django-test-plus` which offers an `APITestCase` and you can just use `self.get(route, ...args)`
like here: 
https://github.com/OpenUpSA/wazimap-ng/blob/3a38bdfb5ea595d1220d0a2667224ec78a29b6cc/tests/profile/test_views.py#L62

Reply:
that's the standard idiom for pytest-drf